extension/src/popup/components/sendPayment/SendTo/index.tsx

```diff
@@ -32,6 +33,23 @@ import {
 
 import "../styles.scss";
 
+const baseReserve = new BigNumber(1);
+
+export const shouldAccountDoesntExistWarning = (
+  isFunded: boolean,
+  assetID: string,
+  amount: string,
+) => {
+  if (
+    !isFunded &&
+    (new BigNumber(amount).lt(baseReserve) ||
+      assetID !== Asset.native().toString())
+  ) {
+    return true;
+  }
+  return false;
+};
```

Comment:
can this just be 
```
export const shouldAccountDoesntExistWarning = (
  isFunded: boolean,
  assetID: string,
  amount: string,
) => (
    !isFunded &&
    (new BigNumber(amount).lt(baseReserve) ||
      assetID !== Asset.native().toString())
  )
```

It should still return the boolean values you're looking for

Reply:
good call https://github.com/stellar/freighter/pull/407/commits/17f8a597599f23da662c476d79d5380db4f252ed